python/sunbeam/parser.py

```diff
@@ -116,3 +117,17 @@ def parse_deck(deck, keywords=[], recovery=[]):
                            # string representation of the the deck.
     pc = _parse_context(recovery) if recovery else lib.ParseContext()
     return lib.parse_deck(deck, keywords, is_file, pc)
+
+
+
+def create_schedule(filename, recovery=[]):
+    if recovery:
+        pc = _parse_context(recovery)
+    else:
+        pc = lib.ParseContext()
+
+    return Schedule(lib.create_schedule(filename, pc))
+
+
+def create_smry_config(filename):
+    return SummaryConfig(lib.create_smry_config(filename))
```

Comment:
I have a feeling that `smry_config` and `schedule` should be private for `parser` and that you instead should call 

```python
state = parse(fname)
sch = state.schedule
smr = state.summary
```

Don't you agree?

Reply:
This was essentially the way it was before; and I agree that was nice. Fundamentally at C++ level the dependencies have been inverted - i.e. the `Schedule` object needs an `EclipseState` object in the construction phase; to achieve your suggestion we would have to implement a `EclipseState.__init__()` method and then assign a `schedule` property there - would probably work but it is more work than the current minumum effort approach. 

Although it could be made private with e.g. use of `_` we need to create a symbol which can be used to invoke creation of a `Schedule` object from python. I see it as valuable and interesting - e.g. for everest - to just be able to construct a Schedule object, that way you could create a state object from one ECLIPSE.DATA file and then mulitple Schedule objects from different Schedule files. 

My suggestion - instead of attaching the `Schedule` object to `EclipseState` object is to create a small structure like:

```Python
class Result(object):

     def __init__(fname):
          self.state = 
          self.schedule = ...
          self.deck = ...
          self.summary_config = .... 
```   
which would be the return value from `sunbeam.parse()`, in addition to (or alternatively) to the individual create methods I have suggested here #63:
```
sunbeam.deck() -> Deck
sunbeam.state() -> EclipseState
sunbeam.summary_config() -> SummaryConfig
sunbeam.schedule() -> Schedule
```